lib/Shippo/ApiRequestor.php

```diff
@@ -155,7 +155,11 @@ private function _interpretResponse($rbody, $rcode)
         }
         return $resp;
     }
-    
+
+    private function _getAuthorizationType($apiKey = '')
```

Comment:
Is there a case where you'd want this to be an empty string?

Reply:
No

Reply:
Line 116 there's a check to see `if (!$myApiKey)` and it'll throw an error. I don't know PHP super well, but my gut tells me that'll pass on an empty string and it should probably throw the same error if its an empty string.

Reply:
Nevermind, PHP does handle this as it should be!